build/run-me.sh

```diff
@@ -101,6 +105,7 @@ OPTIONAL parameters:
                         generated, or a key generated for a previous build in
                         the same work area will be used.
   --downloaddir PATH    Use PATH to store Yocto downloaded sources.
+  --external-dockerfile Specify an external Dockerfile
```

Comment:
Depending on the cross-compile project the Dockerfile will be different?

Reply:
cross-compile needs an external Dockerfile while an external Dockerfile can be used for any project.
Anyway answering to your question, yes, depending the platform we are compiling for the Dockerfile will be different.